Tools/PostProcessing/read_variable_based_adios2.py

```diff
@@ -0,0 +1,145 @@
+import numpy as np
+from adios2 import Stream
```

Comment:
As discussed: we could use [openPMD-api](https://openpmd-api.readthedocs.io/en/0.16.1/usage/examples.html#python).

I think for openPMD-viewer support, we rely on this upcoming feature: https://github.com/openPMD/openPMD-api/pull/1706

Current ADIOS2 support status:
https://openpmd-api.readthedocs.io/en/0.16.1/backends/adios2.html

@franzpoeschel, @guj @pnorbert can you remind me if we can do variable-based encoding without steps (i.e., can we already set something during write that allows-us to read variable based encoding with random access)?

Reply:
> (i.e., can we already set something during write that allows-us to read variable based encoding with random access)

Not yet, that is what #1706 (and follow-ups for changing metadata) will bring (setting a flag during write will not be needed though).
Until that point, it's better to continue using file encoding for random access.